testsuite/tests/prometheus/apicast/selfmanaged/test_batching_caching_policy.py

```diff
@@ -0,0 +1,108 @@
+"""
+Test Prometheus metric for content_caching.
+"""
+from datetime import datetime, timedelta
+import pytest
+
+from packaging.version import Version  # noqa # pylint: disable=unused-import
+from testsuite import TESTED_VERSION, rawobj  # noqa # pylint: disable=unused-import
+from testsuite.gateways.apicast.selfmanaged import SelfManagedApicast
+
+pytestmark = [
+    pytest.mark.skipif("TESTED_VERSION < Version('2.9')"),
+    ]
+
+BATCH_REPORT_SECONDS = 50
+NUM_OF_REQUESTS = 5
+
+
+@pytest.fixture(scope="module")
+def service(service):
+    """Adds policies to servies"""
+    service.proxy.list().policies.append(
+        rawobj.PolicyConfig("3scale_batcher", {"batch_report_seconds": BATCH_REPORT_SECONDS}))
+    return service
+
+
+@pytest.fixture(scope="module")
+def gateway_kind():
+    """Gateway class to use for tests"""
+    return SelfManagedApicast
```

Comment:
Will this work also on OCP 3?

Reply:
no, as test expects to have Prometheus deployed by 3scale monitoring, which is only supported on ocp4.

@ficap @mijaros What do you think about test updating ConfigMap for a template-prometheus (adding new target)? I don't think it's a good idea, so I would keep this test ocp4 only.

Reply:
In that case you can use OperatorApicast instead and limit this test on OCP4 (and skip it with ocp3)

Reply:
suggestion implemented.